third_party/move/tools/move-mutator/src/operators/break_continue.rs

```diff
@@ -0,0 +1,126 @@
+use crate::operator::{MutantInfo, MutationOperator};
+use crate::report::{Mutation, Range};
+use move_command_line_common::files::FileHash;
+use move_compiler::parser::ast;
+use move_compiler::parser::ast::Exp_;
+use std::fmt;
+use std::fmt::Debug;
+
+/// Break and continue mutation operator.
+/// Replaces break and continue statements with each other or deletes them.
+#[derive(Debug, Clone)]
+pub struct BreakContinue {
```

Comment:
Just sharing an idea I'm not sure I even like:
```suggestion
pub struct LoopCtrlStatement {
```

Reply:
I've used a name similar to the names that are in similar tools like SuMo tool for Solidity:
https://github.com/MorenaBarboni/SuMo-SOlidity-MUtator/blob/master/src/operators/standard/break-continue-replacement.js